app/views/steps/international/jurisdiction/edit.html.erb

```diff
@@ -1,23 +1,18 @@
 <% title t('.page_title') %>
+<%= step_header %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
 
-    <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading' %></h1>
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_radio_buttons_fieldset :international_jurisdiction, legend: { tag: 'span', size: 'm' } do %>
```

Comment:
No need for `legend: { tag: 'span', size: 'm' }` and we want the default, header-sized h1 and xl-size here (same in the other views)